aws-lambda/go/README.md

```diff
@@ -0,0 +1,52 @@
+# Steps to test Sentry SDK integration with Node Lambda function:
+
+[Demo Video](https://www.loom.com/share/e3625614df2f44f3bb96286ad7212f99)
+
+## Create a development package
+1. Clone the git repo on your development machine.
+
+2. Build your executable by running
+
+```
+GOOS=linux go build main.go
+```
+
+3. Create a zip
+```
+zip function.zip main
+
+// Include source code in zip to view source code in stacktrace.
+// Or, setup your Sentry<>Github integration instead.
+zip -r function.zip main main.go
+```
+
+## Upload Zip file to Lambda function on AWS.
+
+1. Create a Go lambda function in AWS Lambda.
+2. Function code > Actions > Upload zip
+3. Click 'Test', the request payload is accessible as the second argument in the handler function.
+
+## Usage
+Common to any/all examples below, you'd need to __Configure test events__ for each Lambda function. 
+ * Goto the Lambda function on your AWS Management console
+ * Click *Test* on the far right.
+ * Enter Event name such as MyTestEvent.
+ * Leave the json as is or make it an empty json, your choice :-)
+ * Click Create.
+ * Follow specifics of the examples below.
+ * __Click Test to send your example event to Sentry !!__
+
+## Troubleshooting
+Make sure your lambda Runtime Setting's handler name matches the name of your executable (i.e. main) or your function won't be found when you run the Test.
+
+Setting GOOS to linux ensures that the compiled executable is compatible with the Go runtime, even if you compile it in a non-Linux environment.
```

Comment:
Go runtime -> Lambda runtime.

Same note applies to the architecture as well (GOOS & GOARCH).

Reply:
My note here on the "Setting GOOS to linux.." I found in the AWS documentation and it sounded useful. 

Are you suggesting something like, "Setitng GOOS to linux and GOARCH to amd64 ensures that the compiled executable is compatible with the Go runtime, even if you compile it in a non-Linux environment" ?

Reply:
yes, those two variables control the cross-compilation target. The AWS docs assume the reader is on amd64... which will be incorrect for developers using newer macs, to give one example.

Reply:
thanks, updated on master now.